apps/consumer-client/src/constants.ts

```diff
@@ -1,13 +1,18 @@
 export const PCD_GITHUB_URL = "https://github.com/proofcarryingdata/pcd";
 
 export const IS_PROD = process.env.NODE_ENV === "production";
+export const IS_STAGING = process.env.NODE_ENV === "staging";
 
 export const ZUPASS_URL = IS_PROD
   ? "https://zupass.org/"
+  : IS_STAGING
+  ? "https://staging.zupass.org/"
   : "http://localhost:3000/";
 
 export const PCDPASS_URL = IS_PROD
   ? "https://pcdpass.xyz/"
+  : IS_STAGING
+  ? "https://staging.pcdpass.xyz/"
   : "http://localhost:3000/";
 
 export const ZUPASS_SERVER_URL = IS_PROD
```

Comment:
@ichub I wonder what are the API staging URLs

Reply:
- https://api-staging.pcdpass.xyz/
- https://api-staging.zupass.org/